.github/workflows/test-library.yml

```diff
@@ -195,6 +194,11 @@ jobs:
             && github.event.inputs.release-version
             || steps.scm-version.outputs.dist-version
         }}-py3-none-any.whl')
+    - name: Set PROXYPY_CONTAINER_VERSION env
+      shell: bash
+      run: >-
+        VER=$(echo '${{ needs.pre-setup.outputs.dist-version }}' | tr + .);
```

Comment:
This won't work because `needs` refers to the jobs that you've listed in the `needs:` setting of the current job. And you cannot refer to the outputs before they are created. They will be filled out after this job finishes. Those values are taken from the steps so you can refer to the previous steps by their ids using the `steps` context, the same way the mappings to the outputs are set up later.